wagtail/contrib/search_promotions/templates/wagtailsearchpromotions/list.html

```diff
@@ -5,9 +5,23 @@
     <col />
     <thead>
```

Comment:
A few comments

* There is no way to 'reverse' the ordering of each item, this is something that would be really valuable
* The nesting is a bit odd, would be good to revise and align with other template files
* Using something like `'ohits' in ordering` we can save ourselves some code duplication

See below for a bit of a suggested rework


```
    <thead>
        <tr>
            <th class="title">
                {% trans "Search term(s)" %}
                {% if "query_string" in ordering  %}
                    <a href="{% url 'wagtailsearchpromotions:index' %}?ordering={% if '-' not in ordering %}-{% endif %}query_string" class="icon icon-arrow-{% if '-' in ordering %}up{% else %}down{% endif %}-after teal"></a>
                {% else %}
                    <a href="{% url 'wagtailsearchpromotions:index' %}?ordering=query_string" class="icon icon-arrow-down-after"></a>
                {% endif %}
			</th>
            <th>{% trans "Promoted results" %}</th>
            <th>
                {% trans "Views (past week)" %}
                {% if "ohits" in ordering %}
                    <a href="{% url 'wagtailsearchpromotions:index' %}?ordering={% if '-' not in ordering %}-{% endif %}ohits" class="icon icon-arrow-{% if '-' in ordering %}up{% else %}down{% endif %}-after teal"></a>
                {% else %}
                    <a href="{% url 'wagtailsearchpromotions:index' %}?ordering=ohits" class="icon icon-arrow-down-after"></a>
                {% endif %}
			</th>
        </tr>
    </thead>
```